src/lara.h

```diff
@@ -4684,7 +4684,7 @@ struct Lara : Character {
             if (animIndex == ANIM_RUN_START) {
                 canJump = false;
             } else if (animIndex == ANIM_RUN) {
-                if (frameIndex == 4) {
+                if (frameIndex == 2) {
```

Comment:
it's incorrect, should be 4 for TR2 and probably for newer versions. I thought it was not critical for TR1 (doesn't have this check), if you noticed wrong behavior in TR1 you can modify this condition to
`if ((frameIndex == 4) || ((level->version & TR::VER_VERSION) > TR::VER_TR1)) {`

Reply:
Okay, I'll resubmit once I've fixed it